calculation/gmhazard_calc/gmhazard_calc/verification/GMM/psha_psa_plots.py

```diff
@@ -850,9 +833,7 @@ def pga_vs30_plot(
     period_list = np.array([0.01, 0.1, 1.0, 2.0, 3.0, 5.0])
     rrup = [75, 200]
     # Update the path to the directory to save plots
-    save_path = pathlib.Path(
-        "/home/tom/Documents/QuakeCoRE/resource/verification_plots/special"
-    )
+    save_path = pathlib.Path("put_your_path_here")
```

Comment:
I'd just put '.' to signify the current folder. 

Mind you having an error that says "could not find 'put_your_path_here'" really tells the user what you need to do.
Ignore me, fine as is.

Reply:
That's mind blowing.
I thought I had to do something like `pathlib.Path(__file__)` to point to the current folder.
Or just leave a message so they can set their own directory.